test/framework/src/main/java/org/elasticsearch/transport/nio/AcceptorEventHandler.java

```diff
@@ -94,7 +96,7 @@ public void acceptException(NioServerSocketChannel nioServerChannel, Exception e
      * @param channel that caused the exception
      * @param exception that was thrown
      */
-    public void genericServerChannelException(NioServerSocketChannel channel, Exception exception) {
-        logger.debug("event handling exception", exception);
+    void genericServerChannelException(NioServerSocketChannel channel, Exception exception) {
+        logger.debug(new ParameterizedMessage("exception while handling event for server channel: {}", channel), exception);
```

Comment:
Since this is at the debug level, should you use a supplier here to avoid instantiating the parameterized message if not needed?